FueledUtils/UIExtensions.swift

```diff
@@ -77,6 +97,12 @@ public extension UIView {
 		self.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|[view]|", options: [], metrics: nil, views: views))
 		self.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:|[view]|", options: [], metrics: nil, views: views))
 	}
+
+	public func snapshotImage(afterScreenUpdates: Bool = true) -> UIImage? {
```

Comment:
ah, I see, sorry, confusing diff — I thought it was a method on UIActivityIndicator (which of course I did not understand).